Header header
int16 channel0
int16 channel1
int8  s1
int8  s2
int16 key_code